texts.ts

```diff
@@ -342,9 +342,9 @@ const texts: Text[] = [
     "timeout": [10, 20],
     "texts": [
       {
-        "en": "Hello and welcome aboard. My name is {crewName} and I am the cabin crew member on this flight. As you could hear from our captain - we are currently preparing for departure and we will be taking off shortly. Please make sure that your seatbelt is fastened and your seat back and tray table are in the upright position. Please turn off all personal electronic devices, including laptops and cell phones. Smoking is prohibited for the duration of the flight. If you have any questions or need assistance, please don't hesitate to ask me or one of my colleagues.",
+        "en": "Hello and welcome aboard. My name is {crewName} and I am the purser on this flight. As you could hear from our captain - we are currently preparing for departure and we will be taking off shortly. Please make sure that your seatbelt is fastened and your seat back and tray table are in the upright position. Please turn off all personal electronic devices, including laptops and cell phones. Smoking is prohibited for the duration of the flight. If you have any questions or need assistance, please don't hesitate to ask me or one of my colleagues.",
```

Comment:
Hi! I'm not sure if "pursuer" is the right word here. To be honest, I've never heard this in real life.

Reply:
Hi,

Thanks for your reply. When posting this, I’ve just realized that this section could potentially be misunderstood.

If the flight attendant were to address the cabin this way, it might give the impression that he or she is the *only* cabin crew member on the flight ("...and I am the cabin crew member").

Would it be possible to change the phrase "and I am the cabin crew member" to "lead flight attendant"? 

That would make it clearer. :)